src/components/inline/Button/Button.js

```diff
@@ -1,20 +1,58 @@
+import { Link } from "gatsby";
 import PropTypes from "prop-types";
 import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ children }) => (
-  <button className={s.button} type="button">
-    {children}
-  </button>
-);
-
-Button.propTypes = {
-  children: PropTypes.node,
+const Button = ({ button }) => {
```

Comment:
Is it possible to set the props of this component such that you can pass them in as a flat list rather than in this `button` prop? For example, as a user of this component, I think I would prefer to write

```jsx
<Button text="Click Me" externalLink="https://google.com" />
```

instead of

```jsx
<Button button={{ title: "Click Me", externalLink: "https://google.com"}} />
```